src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -63,17 +67,20 @@ namespace alpha_complex {
  * 
  * Please refer to \ref alpha_complex for examples.
  *
- * The complex is a template class requiring an Epick_d <a target="_blank"
+ * The complex is a template class requiring an <a target="_blank"
+ * href="https://doc.cgal.org/latest/Kernel_d/structCGAL_1_1Epeck__d.html">CGAL::Epeck_d</a>,
+ * or an <a target="_blank"
+ * href="https://doc.cgal.org/latest/Kernel_d/structCGAL_1_1Epick__d.html">CGAL::Epick_d</a> <a target="_blank"
```

Comment:
The same sentence is present in the user manual and should be updated there as well.

Reply:
You are right. I added it here : c8be137d15f40f40c4150f0c0fb9776cd680a36c
I also mentionned the CGAL 5.X performance here.